docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -276,6 +276,19 @@ You can also check the GPUs available on your nodes by running:
     my-cluster-5               H100      8           8
 
 
+Setting up NFS and other volumes
```

Comment:
Add to "Setting up Kubernetes cluster for SkyPilot" bullet list above?

Reply:
Added and reorganized the optional steps